### ISSUE TITLE
View change: not enough proofs

### ISSUE THREAD

**Issue by Gilthoniel** [dedis/cothority#2127]
Sometimes the view change gets stuck with:
```
E 19/29/10 15:23:12.138834793: viewchange.go:336 (byzcoin.(*Service).verifyViewChange) - tls://fairywren.ch:7770 not enough proofs: %v <= %v 2 2
```

**Comment by ineiti** [comment]
I started #2126 to get more info why and when this happens. I also tried to reproduce it, but it's a bit difficult. The situation we were in on Tuesday, 29th of October, was the following:
- all nodes were stopped
- nodes came trickeling in over 30 minutes and tried to view-change because the leader was not there
- the latest leader (the one in the latest block) came online after all other nodes
- every viewchange failed with the above message, it having only 2 signatures, even though it should have more

**Comment by ineiti** [comment]
In fact this has been solved by #2255
